main.py

```diff
@@ -1,10 +1,40 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as f:
+        players_info = json.load(f)
+
+    for pl in players_info.keys():
```

Comment:
Consider using `players_info.items()` instead of `players_info.keys()` to directly access both the key and value in the loop, which can make the code cleaner and more efficient.